content/ecosystem/social-channels/index.en.yaml

```diff
@@ -316,19 +363,19 @@ content:
       key: website ETC Hub
       link: https://etchub.io/
       name: ETCHub.io
-    -
-      key: website EthereumClassicAsia
-      link: http://ethereumclassicasia.org/
-      name: EthereumClassicAsia.org
+      brand: etc
```

Comment:
I don't think this will work yet; there isn't an icon programmed in for etc yet.

But i'll address this soon

Reply:
Great, that was my thinking when I added it. That we could add the logos that did not work, like YouTube, Matrix, and have our own ETC